libs/core/algorithms/include/hpx/parallel/algorithms/uninitialized_default_construct.hpp

```diff
@@ -11,8 +11,8 @@
 #if defined(DOXYGEN)
 namespace hpx {
 
-    /// Constructs objects of type typename iterator_traits<ForwardIt>
-    /// ::value_type in the uninitialized storage designated by the range
```

Comment:
The `::value_type` part needs to stay, but the whole `typename...` should be on a single line to appease doxygen.

Reply:
Ok!